accesscontrol/jwt.go

```diff
@@ -113,6 +109,18 @@ func NewJWT(options *JWTOptions) (*JWT, error) {
 
 // Validate reading the token from configured source and validates against the key.
 func (j *JWT) Validate(req *http.Request) error {
+	ctx := req.Context()
+	cctx, ok := req.Context().Value(request.ContextType).(content.Context)
+	if !ok {
+		return fmt.Errorf("HTTP context must have content.Context")
```

Comment:
This will never happen with calls via server handler. Anyway, a specific error type should be used since this would fallback to an auth error which is wrong.